site/pages/add-consequence.tsx

```diff
@@ -0,0 +1,144 @@
+import { NextPageContext } from "next";
+import { destroyCookie, parseCookies } from "nookies";
+import { ReactElement } from "react";
+import Radios from "../components/form/Radios";
+import { TwoThirdsLayout } from "../components/layout/Layout";
+import { COOKIES_ADD_CONSEQUENCE_ERRORS, COOKIES_ADD_CONSEQUENCE_INFO } from "../constants/index";
+import { InputInfo, ErrorInfo } from "../interfaces/index";
+
+interface AddConsequenceWithErrors {
+    errors: ErrorInfo[];
+    inputs: AddConsequenceProps;
+}
+
+export interface AddConsequenceProps {
+    modeOfTransport?: TransportMode;
+    consequenceType?: ConsequenceType;
+}
+
+export enum TransportMode {
+    bus = "Bus",
+    tram = "Tram",
+    ferry = "Ferry",
+    train = "Train",
+}
+
+export enum ConsequenceType {
+    services = "Services",
+    networkWide = "Network Wide",
+    operatorWide = "Operator Wide",
+    stops = "Stops",
+}
+
+const title = "Add a Consequence";
```

Comment:
Titles and text are almost always sentence case, which means this needs to become 'Add a consequence'

Reply:
Although, the usual standard we use is as below

```
const title = "Create Disruptions";
const description = "Create Disruptions page for the Create Transport Disruptions Service";
```

so this new one would be 

```
const title = "Create Consequences";
const description = "Create Consequences page for the Create Transport Disruptions Service";
```